PureBasicIDE/Common.pb

```diff
@@ -1547,6 +1547,14 @@ EndEnumeration
 
 #MATCH_Exact  = #MATCH_OS|#MATCH_Version|#MATCH_Beta|#MATCH_Processor
 
+;
+;- Values for SaveProjectSettings preference
+;
+#SAVESETTINGS_EndOfFile  = 0
+#SAVESETTINGS_PerFileCfg = 1
+#SAVESETTINGS_CommonCfg  = 2
```

Comment:
The IDE display "A common file project.cfg for every directory". To  match the name of the previous constant you shoul name it `#SAVESETTINGS_PerFolderCfg".
I think the "project" word should be keep away since it's unrelated to PureBasic project.

Reply:
> Edit : I don't know why GitHub select 4 lines. this comment is only for line 1555.

Because diff changes come in [hunks].

[hunks]: https://www.gnu.org/software/diffutils/manual/html_node/Hunks.html